### ISSUE TITLE
test_reloc failing on x86

### ISSUE THREAD

**Issue by hainest** [dyninst/dyninst#714]
I'm seeing this on the new [coriander](https://bottle.cs.wisc.edu/details/499) with Kernel 5.0 and glibc-2.29. @xmeng reported failures on [gpu.cs.rice.edu](https://github.com/dyninst/dyninst/pull/638#issuecomment-591657029). I will continue to test.

**Comment by mxz297** [comment]
gpu.cs.rice.edu has the following spec:

```
os: Linux
hostname: gpu.cs.rice.edu
kernel: 3.10.0-862.11.6.el7.x86_64
version: #1 SMP Fri Aug 10 16:55:11 UTC 2018
arch: x86_64/GenuineIntel
libc: 2.17
date: 2020-02-27T15:16:01.
```

**Comment by hainest** [comment]
Merged into #722. Closing.
